app/config/ConfigKeys.scala

```diff
@@ -42,6 +42,7 @@ object ConfigKeys {
   val useDirectDebitDummyPageFeature: String = "features.useDirectDebitDummyPage.enabled"
   val useLanguageSelectorFeature: String = "features.useLanguageSelector.enabled"
   val submitReturnFeatures: String = "features.submitReturnFeatures.enabled"
+  val checkMandationStatus: String = "features.checkMandationStatus.enabled"
```

Comment:
Please can you delete this as this feature switch already exists. It's the submitReturnFeatures one above.